routes/routes.ts

```diff
@@ -328,9 +322,7 @@ router.get("/categories", async (req: Request, res: Response) => {
           console.log(
             `error accessing categories \n ${errorObject.name} : ${errorObject.message}`
           );
-          return res
-            .status(409)
-            .send("There was an error accessing your categories");
+          ResponseHandler.clientError(res, "There was an error accessing your categories", 409);
```

Comment:
The error message here for the client error response gives meaningful info. I think we should also replicate this for success messages

Reply:
as well as server error messages